Dockerfile-ubuntu.template

```diff
@@ -205,7 +205,11 @@ RUN set -eux; \
 	\
 # Check that OpenSSL still works after copying from previous builder
 	ldconfig; \
+	sed -i.ORIG -e '/\.include.*fips/s/.*/.include \/usr\/local\/etc\/ssl\/fipsmodule.cnf/' \
+	    -e '/# fips =/s/.*/fips = fips_sect/' /usr/local/etc/ssl/openssl.cnf; \
+	sed -i.ORIG -e '/^activate/s/^/#/' /usr/local/etc/ssl/fipsmodule.cnf; \
```

Comment:
These are a little fiddly and moderately concerning, but I trust that you're going to be actively on top of maintaining them so I'm not too worried about it. :smile:

Reply:
Yeah, it is fiddly. I left the `.ORIG` files in case a user gets curious and pokes around the image a bit.

Once Erlang/OTP and RabbitMQ have switches to turn on FIPS I'll re-visit this section of the build process.